helix-lsp/src/transport.rs

```diff
@@ -215,20 +215,15 @@ impl Transport {
             }
         };
 
-        let tx = self
-            .pending_requests
-            .lock()
-            .await
-            .remove(&id)
-            .expect("pending_request with id not found!");
-
-        match tx.send(result).await {
-            Ok(_) => (),
-            Err(_) => error!(
-                "Tried sending response into a closed channel (id={:?}), original request likely timed out",
-                id
-            ),
-        };
+        if let Some(tx) = self.pending_requests.lock().await.remove(&id) {
+            match tx.send(result).await {
+                Ok(_) => (),
+                Err(_) => error!(
+                    "Tried sending response into a closed channel (id={:?}), original request likely timed out",
+                    id
```

Comment:
```suggestion
                    "Tried sending response into a closed channel (id={id}), original request likely timed out",
```

Reply:
We should avoid raising the MSRV, I think.

Reply:
I thought helix is building on stable rust?

Reply:
Oh https://github.com/helix-editor/helix/pull/1881

I thought we should have another build in CI with the minimum rust version.